Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.6-slim
 ENV RABBITMQ_HOST rabbitmq
```

Comment:
`RABBITMQ_HOST ` isn't used anywhere else in the repo, same goes with `RABBITMQ_PORT`. Think it should be `RABBITMQ_SERVICE_HOST` and `RABBITMQ_SERVICE_PORT`

Reply:
Cheers, I began changing it everywhere after another comment, then decided not too.